src/utils/ddb/SConscript

```diff
@@ -5,9 +5,6 @@ def scons():
     """Execute build"""
     Import('env', 'prereqs')
 
-    if not prereqs.server_requested():
```

Comment:
why remove this check?

Reply:
It is done in src/utils/SConscript already